src/mpid/ch4/generic/am/mpidig_am.h

```diff
@@ -64,6 +65,13 @@ enum {
     MPIDIG_HANDLER_STATIC_MAX
 };
 
+enum {
+    MPIDIG_RNDV_GENERIC = 0,
+    MPIDIG_RNDV_IPC,
```

Comment:
Is this crossed over from your IPC PR? I would say add the enum in the IPC PR rather than at the introduction of the rndv cb framework.

Reply:
I find it odd to introduce an enum with only a single value. Sure, I'll fix.

Reply:
fixed.